pkg/lister/testing/integration_test_helper.go

```diff
@@ -0,0 +1,96 @@
+//go:build integration
+
+package testing
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+	fn "knative.dev/func/pkg/functions"
+	"knative.dev/func/pkg/oci"
+	fntest "knative.dev/func/pkg/testing"
+	fnk8stest "knative.dev/func/pkg/testing/k8s"
+)
+
+func IntegrationTest(t *testing.T, lister fn.Lister, deployer fn.Deployer, describer fn.Describer, remover fn.Remover, deployerName string) {
```

Comment:
I think we should also add a call to `t.Helper()` as the first line of all these abstracted test functions.